scheduler/src/cook/mesos/api.clj

```diff
@@ -67,10 +67,8 @@
   [conn jobs :- [Job]]
   (doseq [{:keys [uuid command max-retries max-runtime priority cpus mem user name ports uris env]} jobs
           :let [id (d/tempid :db.part/user)
-                ports (mapv (fn [port]
-                              ;;TODO this schema might not work b/c all ports are zero
-                              [:db/add id :job/port port])
-                            ports)
+                ports (when (and ports (not (zero? ports)))
```

Comment:
Why do we discount the port if it is zero? Simply because we default to it later? Why does everything get a port?